centreon/www/install/php/Update-22.04.8.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+/*
+ * Copyright 2005 - 2022 Centreon (https://www.centreon.com/)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * For more information : contact@centreon.com
+ *
+ */
+
+require_once __DIR__ . '/../../class/centreonLog.class.php';
+$centreonLog = new CentreonLog();
+
+//error specific content
+$versionOfTheUpgrade = 'UPGRADE - 22.04.8: ';
+$errorMessage = '';
+
+try {
+    $errorMessage = "Unable to remove app_key column";
+        $pearDB->beginTransaction();
+        $pearDB->query("ALTER TABLE `remote_servers` DROP COLUMN `app_key`");
```

Comment:
Indentation is not ok
```suggestion
    $pearDB->beginTransaction();
    $pearDB->query("ALTER TABLE `remote_servers` DROP COLUMN `app_key`");
```